python/paddle/fluid/layers/tensor.py

```diff
@@ -378,6 +379,97 @@ def concat(input, axis=0, name=None):
     return out
 
 
+def broadcast_tensors(input, name=None):
```

Comment:
why put api into fluid, move it to paddle/tensor/manipulation.py, fluid no need to add new api